.evergreen/config.yml

```
@@ -146,12 +146,9 @@ functions:
         script: |
           ${PREPARE_SHELL}
           rm -rf $DRIVERS_TOOLS
-          if [ "${project}" = "drivers-tools" ]; then
-            # If this was a patch build, doing a fresh clone would not actually test the patch
-            cp -R ${PROJECT_DIRECTORY}/ $DRIVERS_TOOLS
-          else
-            git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
-          fi
+          git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
+          # Pin drivers-evergreen-tools to revision 98f6b0e (Aug 20, 2025).
+          git -C $DRIVERS_TOOLS checkout 98f6b0e
```

[nitpick] We should have a prescribed way of cherry-picking changes to DET required to resolve problems occurring due to server changes. Suggest adding a comment:
```suggestion
		 # In leu of cutting a dedicated branch, please cherry-pick any required changes inline.
          git -C $DRIVERS_TOOLS checkout 98f6b0e
```

---

Since this is for v1, we don't need to support any server versions after MongoDB 8.0.

How would cherry-picking DET changes inline work?

---

@matthewdale 

> we don't need to support any server versions after MongoDB 8.0.

Then we should remove the latest test matrices from the evergreen configuration for v1

> How would cherry-picking DET changes inline work?

```
git -C $DRIVERS_TOOLS checkout 98f6b0e
git cherry-pick <sha>
```

---

Both of those suggestions make sense. I'll remove the "latest" tasks and add the comment you suggested.